tests/integrations/config/mod.rs

```diff
@@ -677,6 +677,8 @@ fn test_serde_custom_tikv_config() {
         region_split_keys: 100000,
         consistency_check_method: ConsistencyCheckMethod::Raw,
         perf_level: PerfLevel::EnableTime,
+        enable_region_bucket: false,
+        region_bucket_size: ReadableSize::gb(1),
```

Comment:
Why the config default value(1G) is deferrent from DEFAULT_BUCKET_SIZE(96M).

Reply:
It tests whether the config is serialized and deserialized correctly, so its value should be different.